src/Microsoft.AspNet.Server.Kestrel/Infrastructure/MemoryPoolIterator2.cs

```diff
@@ -228,18 +228,13 @@ public int Seek(int char0)
             }
         }
 
-        public int Seek(int char0, int char1)
+        public int Seek(Vector<byte> byte0Vector, Vector<byte> byte1Vector)
```

Comment:
Why not continue using `ch0Vector`, `ch0Equals`, `ch0Index` names?

Makes this PR a little easier to review.